packages/wallets/src/features/accounts/screens/ResubmitPOA/ResubmitPOA.scss

```diff
@@ -95,4 +99,116 @@
             }
         }
     }
+
+    &__document {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 2.4rem;
+        max-width: 85rem;
+
+        @include mobile {
+            width: 100%;
+        }
+
+        &__title {
+            display: flex;
+            height: 2.4rem;
+            justify-content: center;
+            align-items: center;
+            gap: 1.1rem;
+
+            @include mobile {
+                display: flex;
+                align-items: center;
+                gap: 8px;
+                align-self: stretch;
+            }
+
+            &__divider {
+                width: 66rem;
+                height: 0.12rem;
+                flex-shrink: 0;
+                background: var(--system-light-7-secondary-background, #f2f3f4);
+
+                @include mobile {
+                    height: 0.1rem;
+                    flex: 1 0 0;
+                }
+            }
+        }
+
+        &__container {
+            display: flex;
+            padding: 1.6rem 2.4rem;
+            flex-direction: column;
+            align-items: flex-start;
+            gap: 1.6rem;
+            align-self: stretch;
+            border-radius: 0.8rem;
+            border: 0.1rem solid var(--system-light-5-active-background, #d6dadb);
+
+            @include mobile {
+                border: none;
+                padding: 0;
+            }
+
+            &__disclaimer {
+                display: flex;
+                flex-direction: column;
+                align-items: flex-start;
+                gap: 1.6rem;
+                align-self: stretch;
+
+                &__list {
+                    list-style-type: disc;
+                    padding-left: 2rem;
+                }
+            }
+
+            &__upload {
+                display: flex;
+                flex-direction: column;
+                align-items: flex-start;
+                gap: 1.6rem;
+                align-self: stretch;
+
+                & > .wallets-dropzone__container {
+                    display: flex;
+                    flex-direction: column;
+                    justify-content: center;
+                    align-items: center;
+                    gap: 1.6rem;
+                    align-self: stretch;
+                    border-radius: 0.8rem;
+                    border: 0.1rem dashed var(--system-light-5-active-background, #d6dadb);
+                    cursor: pointer;
+                }
+
+                & .wallets-dropzone__placeholder {
+                    gap: 2rem;
+                }
+
+                & .wallets-dropzone__placeholder-icon {
+                    width: auto;
+                    height: auto;
+                }
```

Comment:
not really needed because by default its already auto namsayin 🤷🏼

Reply:
yes but im overwriting it. Width is defined in the dropzone component